GoogleForCreators/web-stories-wp
# Floating Menu: Shapes background color picker

## Context


The floating menu for Shapes still had placeholder button for the color picker, eyedropper and opacity. We are just added that here. 

## Summary
This pr is blocked (branched off of) by `sidebar`. We need the update to the color popup in order for the positioning to be correct. 


## Relevant Technical Choices
Looks like the shared `color` component for all the floating menus has a `width` prop that was being sent to `Color`, but it was never being used. I hooked that up so that the width of the inputs can be set dynamically based on those inputs. 


## To-do
n/a


## User-facing changes
<img width="788" alt="Screen Shot 2022-03-16 at 12 38 30 PM" src="https://user-images.githubusercontent.com/1820266/158663230-b49d06f5-1110-4109-af4b-f55318fd73db.png">



## Testing Instructions






This PR can be tested by following these steps:

1. Open floating menu for a shape element
2. check that changing color and opacity works as expected and is synced with design panel. 


## Reviews

### Does this PR have a security-related impact?
no


### Does this PR change what data or activity we track or use?
I added tracking (to be in sync with other floating menu tracking) 


### Does this PR have a legal-related impact?
no


## Checklist




---



Fixes #10933 
Fixes: #10934
